package.json

```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "start": "webpack-dev-server",
     "build": "webpack",
-    "test": "react-scripts test --env=jsdom",
+    "test": "jest",
     "eject": "react-scripts eject",
```

Comment:
I think we can remove `react-scripts` dependency entirely from the project at this point.